dist/Net-Ping/Makefile.PL

```diff
@@ -0,0 +1,75 @@
+use strict;
+use warnings;
+use v5.002; # not using our in the CPAN release
+use ExtUtils::MakeMaker;
+
+my @extras = ();
+my $EUMM_VER = $ExtUtils::MakeMaker::VERSION;
+my @AUTHORS = (
+      'Reini Urban <rurban@cpan.org>',
+      'Steve Peters <steve@fisharerojo.org>',
+      'Matthew Musgrove <mr.muskrat@gmail.com>',
+      'Karl Williamson <khw@cpan.org>',
+      'Brian Fraser <fraserbn@gmail.com>',
+      'Mark Gardner <mjgardner@cpan.org>',
+  );
+
+push @extras,
+  AUTHOR => join(", ", @AUTHORS)
+  if $EUMM_VER gt '5.4301' and $EUMM_VER lt '6.57_02';
+push @extras,
+  AUTHOR => [ @AUTHORS ]
+  if $EUMM_VER ge '6.57_02';
+push @extras, SIGN => 1
+  if $EUMM_VER ge '6.18';
+push @extras, LICENSE => 'perl_5'
+  if $EUMM_VER ge '6.31' and $EUMM_VER le '6.46';
+push @extras,
+  META_MERGE => {
+      'meta-spec' => { version => 2 },
+        resources   => {
+            # TODO: 26 old issues still open at RT
+            # https://rt.cpan.org/Public/Dist/Display.html?Name=Net-Ping
+            bugtracker  => 'https://github.com/rurban/Net-Ping/issues',
+            repository  => {
+                type => 'git',
+                url => 'https://github.com/rurban/Net-Ping.git',
+                web => 'https://github.com/rurban/Net-Ping',
+            },
```

Comment:
These should point to the perl5 github if we're upstream

Reply:
Fixed, also made the INSTALLDIRS consistent with the others.